_posts/es/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,262 @@
+  vulnerable.
+
+- **Maleabilidad de la firma Tapscript:** en un aparte de la mencionada
+  conversación sobre la dilución de tasas, el desarrollador Russell O'Connor
+  [señaló][o'connor tsm] que las firmas para un [tapscript][tema
+  tapscript] pueden aplicarse a una copia del tapscript colocada en otro lugar
+  en el árbol taproot.  Por ejemplo, el mismo tapscript *A* se coloca en
+  dos lugares diferentes en un árbol taproot.  Para utilizar la alternativa más profunda
+  requerirá colocar un hash adicional de 32 bytes en los datos testigo de
+  la transacción de gasto.
+
+  ```text
+    *
+  / \
+  A   *
+    / \
+    A   B
+  ```
+
+  Eso significa que incluso si Mallory proporciona a Bob un testigo válido para
+  su gasto tapscript antes de que Bob proporcione su propia firma, es
+  todavía posible para Mallory emitir una versión alternativa de la
+  transacción con un testigo mayor. Bob sólo puede evitar este problema
+  recibiendo de Mallory una copia completa de su árbol de tapscripts.
+
+  En el contexto de futuras actualizaciones mediante un soft fork de Bitcoin, Anthony Towns
+  abrió un [issue][bitcoin inquisition #19] en el repositorio Bitcoin
+  Inquisition siendo utilizado para probar [SIGHASH_ANYPREVOUT][tema
+  sighash_anyprevout] (APO) para considerar tener un commit de APO para agregar datos adicionales y así evitarles este problema a los usuarios de esa extensión.
+
+## Cambios en servicios y software cliente
+
+*En este artículo mensual, destacamos actualizaciones interesantes de wallets (billeteras/ carteras/ monederos) y servicios de Bitcoin.*
+
+- **El monedero Liana añade multisig:**
+[Liana][news234 liana]'s [0.2 release][liana 0.2] añade soporte multisig usando
+[descriptores][topic descriptors].
+
+- **Lanzamiento de la cartera Sparrow 1.7.2:**
+La [versión 1.7.2][sparrow 1.7.2] de Sparrow añade soporte para [taproot][topic taproot]
+soporte, [BIP329][] características de importación y exportación (ver [Newsletter #235][news235
+bip329]), y soporte adicional para dispositivos de firma por hardware.
+
+- **La biblioteca Bitcoinex añade compatibilidad con schnorr:**.
+[Bitcoinex][bitcoinex github] es una librería de utilidades Bitcoin para el lenguaje de programación funcional Elixir.
+
+- **Libwally 0.8.8 publicado:**
+[Libwally 0.8.8][] añade soporte para hash etiquetado [BIP340][], soporte adicional para sighash
+incluyendo [BIP118][] ([SIGHASH_ANYPREVOUT][topic SIGHASH_ANYPREVOUT]), y
+funciones adicionales [miniscript][tema miniscript], descriptor y [PSBT][tema psbt].
+
+## Optech Recomienda
+
+[BitcoinSearch.xyz][] es un motor de búsqueda recientemente lanzado para Bitcoin
+documentación técnica y debates.  Se utilizó para encontrar rápidamente
+varias de las fuentes enlazadas en este boletín, una gran mejora
+sobre otros métodos más laboriosos que hemos utilizado anteriormente.  Contribuciones
+a su [código][repositorios bitcoinsearch] son bienvenidas.
```

Comment:
this is an example of reference that doesn't match the one at the bottom of the page
```suggestion
a su [código][bitcoinsearch repos] son bienvenidas.
```